modules/Bio/EnsEMBL/Compara/Utils/JIRA.pm

```diff
@@ -77,8 +77,17 @@ sub new {
     $self->{_user} = $user || $ENV{'USER'};
     $self->{_relco} = $relco || $self->{_user};
     $self->{_project} = $project || 'ENSCOMPARASW';
-    # Initialise user's password
-    $self->{_password} = defined $ENV{'USERPW'} ? $ENV{'USERPW'} : $self->_request_password();
+
+    if ( $ENV{'JIRA_AUTH_TOKEN'} ) {
```

Comment:
```suggestion
    if ( exists $ENV{'JIRA_AUTH_TOKEN'} && defined $ENV{'JIRA_AUTH_TOKEN'} ) {
```
(or maybe you prefer to raise an error if the env variable is not defined)

Reply:
> (or maybe you prefer to raise an error if the env variable is not defined)

Yes, that's what this does 😉  Don't think we need this suggested change either.